src/components/Footer/Footer.js

```diff
@@ -0,0 +1,56 @@
+import './Footer.css';
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+import messages from '../../assets/local/en';
+import { contacts } from '../../assets/contacts';
+
+const local = messages.footer;
+
+function Footer() {
+  return (
+    <footer className="footer">
+      <div className="footer__info">
+        <div className="footer__contacts">
+          <div className="footer__contact">
+            <strong> {local.address}</strong>: {contacts.address}
+          </div>
+          <div className="footer__contact">
+            <strong> {local.phone}</strong>:{' '}
+            <a className="footer__link" href={`tel:${contacts.phone}`}>
+              {contacts.phone}
+            </a>
+          </div>
+          <div className="footer__contact">
+            <strong> {local.email}</strong>:{' '}
+            <a className="footer__link" href={`mailto: ${contacts.email}`}>
+              {contacts.email}
+            </a>
+          </div>
+          <div className="footer__contact">
+            <strong>
+              <a className="footer__link" href={contacts.linkedIn}>
+                LinkedIn
+              </a>
+            </strong>
+          </div>
+        </div>
+        <div className="footer__copyright">{contacts.copyright}</div>
```

Comment:
I think that there would be a better semantic tag for this. The `<small>` tag would be appropriate.

Reply:
Font size for copyright is the same as for contacts(16px). I can add this tag for semantic, sure.

Reply:
I used <p> instead of <small>. Tag <small> makes the text smaller, but according to Figma design, contacts and copyright have the same font size.